lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -231,18 +231,18 @@ class TestFieldOverrideMongoPerformance(FieldOverridePerformanceTestCase):
         #     # of sql queries to default,
         #     # of mongo queries,
         # )
-        ('no_overrides', 1, True, False): (25, 1),
-        ('no_overrides', 2, True, False): (25, 1),
-        ('no_overrides', 3, True, False): (25, 1),
-        ('ccx', 1, True, False): (25, 1),
-        ('ccx', 2, True, False): (25, 1),
-        ('ccx', 3, True, False): (25, 1),
-        ('no_overrides', 1, False, False): (25, 1),
-        ('no_overrides', 2, False, False): (25, 1),
-        ('no_overrides', 3, False, False): (25, 1),
-        ('ccx', 1, False, False): (25, 1),
-        ('ccx', 2, False, False): (25, 1),
-        ('ccx', 3, False, False): (25, 1),
+        ('no_overrides', 1, True, False): (26, 1),
```

Comment:
Reviewers: the Waffle queries appear to have added one or two queries to certain tests on the course info page. Since that page is going away, I don't feel like this is a concern. Let me know if you think otherwise.